governance/polls/Investigate Offboarding of Multiple Vault Types - June 21, 2021

```diff
@@ -0,0 +1,55 @@
+---
+title: Investigate Offboarding of Multiple Vault Types - June 21, 2021
+summary: Signal your support or opposition to investigate offboarding of multiple vault types.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628
+vote_type: Plurality Voting
+categories:
+   - Collateral
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Investigate on Offboarding of multiple collateral types - June 21, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, June 21 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the Mandated Actors should investigate a plan to offboard the following vault types:
+
+* USDT-A
+* UNIV2-AAVEETH-A
+* KNC-A
+* UNIV2-LINKETH-A
+* LRC-A
+* ZRX-A
+* BAT-A
+* MANA-A
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following action will be taken:**
+* The Mandated Actors will investigate a plan to offboard the above vault types from the Maker Protocol. 
+* A future executive will ask the community to implement a proposal from the Mandated Actors.
```

Comment:
Should probably read 'A future on-chain poll will ask the community to implement a specific proposal from the Mandated Actors'

I don't think we'd want to go straight to executive without confirming the plan.